CONTRIBUTING.md

```diff
@@ -1,9 +1,15 @@
 Contributing
 ============
-We gratefully accept bug reports and contributions from the community. There are some requirements we need to fulfill in order to be able to integrate contributions:
+We gratefully accept bug reports and contributions from the community.
 
- - As with any open source project, contributions will be reviewed by the project team and community and may need some modifications to be accepted.
```

Comment:
Please keep the part about reviews. It's typically the most visible aspect of contributing to Mbed TLS, so it deserves prominence.